E/Elfutils/build_tarballs.jl

```diff
@@ -1,29 +1,46 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
-using BinaryBuilder
+using BinaryBuilder, Pkg
 
 name = "Elfutils"
-version = v"0.177"
+version = v"0.179"
 
 # Collection of sources required to build Elfutils
 sources = [
     ArchiveSource("https://sourceware.org/elfutils/ftp/$(version.major).$(version.minor)/elfutils-$(version.major).$(version.minor).tar.bz2",
-                  "fa489deccbcae7d8c920f60d85906124c1989c591196d90e0fd668e3dc05042e"),
+                  "25a545566cbacaa37ae6222e58f1c48ea4570f53ba991886e2f5ce96e22a23a2"),
+    DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/elfutils-*/
+if [[ ${target} = *-musl* ]] ; then
+    for patchfile in $WORKSPACE/srcdir/patches/*; do
+        atomic_patch -p1 $patchfile
+    done
+    cp $WORKSPACE/srcdir/error.h src/
+    cp $WORKSPACE/srcdir/error.h lib/
+
+    # For some reason, just installing bsd-compat-headers doesn't work
```

Comment:
I don't think `/usr/include` is a search path for the cross compilers

Reply:
Updated the comment appropriately